manifests/plugin/pulp/params.pp

```diff
@@ -8,11 +8,11 @@
   $version            = undef
   $group              = undef
   $pulpnode_enabled   = false
-  $pulp3_enabled      = false
-  $pulp3_mirror       = false
+  $pulpcore_enabled      = false
```

Comment:
the indentation of the `=` is now inconsistent in this file

Reply:
Thanks,